README.md

```diff
@@ -132,6 +133,24 @@ Releases to PyPI is an Github Action which needs to be manually triggered.
 [![Release](https://github.com/spetlr-org/spetlr/actions/workflows/release.yml/badge.svg)](https://github.com/spetlr-org/spetlr/actions/workflows/release.yml)
 [![PyPI spetlr badge](https://img.shields.io/pypi/v/spetlr)](https://pypi.org/project/spetlr/)
 
+
+# Requirements and dependencies
+The library has three txt-files at the root of the repo. These files defines three levels of requirements:
+- `requirements_install.txt` - this file contains the required libraries to be able to install spetlr.
+- `test_requirements.txt` - libraries required to run unit- and integration tests
+- `dev_requirements.txt` - libraries required in the development process in order to contribute to the repo
+
+All libraries and their dependencies are added with a fixed version to the configuration file `setup.cfg`.
+
+To __upgrade__ the the dependencies in the `setup.cfg` file do the following:
+
+1. Create a new branch
+2. Run `upgrade_requirements.ps1` in your terminal
+3. Commit the changes the script has made to the cfg file. If there are no changes everything is up to date.
+4. The PR runs all tests and ensure that the library is compliant with any updates
+
+Note that if it is desired to upgrade a dependency, but not to its newest version, it is possible to set the desired version in the `install_requirements.txt`, then this will be respected by the upgrade script.
```

Comment:
Again, it is unclear whether the requirements in `setup.cfg` and `install_requirements.txt` should be, and if yes, are aligned.

Reply:
Resolving as my comment is outdated.